src/components/Dialog/Dialog.tsx

```diff
@@ -38,6 +38,7 @@ interface DialogOwnProps {
     disableFocusTrap?: boolean;
     disableAutoFocus?: boolean;
     restoreFocusRef?: React.RefObject<HTMLElement>;
+    scroll?: 'inner' | 'outer';
```

Comment:
I suggest this naming instead: `contentOverflow?: 'visible' | 'auto'`, symmetrical to the CSS overflow property